src/actions/upload.js

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable camelcase */
-import client from 'part:@sanity/base/client'
 import {uuid as generateUuid} from '@sanity/uuid'
 import {isString} from 'lodash'
-import {throwError, of, from, concat, defer} from 'rxjs'
-import {mergeMap, catchError, mergeMapTo, switchMap} from 'rxjs/operators'
-import studioClient from 'part:@sanity/base/client'
-import {createUpChunkObservable} from '../clients/upChunkObservable'
+import {default as client, default as studioClient} from 'part:@sanity/base/client'
```

Comment:
What is the reason for importing `default` twice here under two different names?

Reply:
Good catch! No reason, must've been mid-refactoring and gotten sidetracked! 

This is fixed in 538c124